test/functional/collection_tests.js

```diff
@@ -12,65 +11,60 @@ describe('Collection', function() {
     return setupDatabase(this.configuration);
   });
 
-  /**
-   * @ignore
-   */
-  it('should correctly execute basic collection methods', {
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
-
-    // The actual test we wish to run
-    test: function(done) {
-      var self = this;
-      var client = self.configuration.newClient(self.configuration.writeConcernMax(), {
+  describe('(non >2.1.0)', function() {
+    let client;
+    let db;
+    beforeEach(function() {
```

Comment:
You could extract out the closing of the client to an `afterEach` since it happens in all of the tests here

Reply:
Fixed